src/coreclr/jit/morph.cpp

```diff
@@ -3634,7 +3655,7 @@ void Compiler::fgInitArgInfo(GenTreeCall* call)
 #ifdef WINDOWS_AMD64_ABI
                             // Whenever we pass an integer register argument
                             // we skip the corresponding floating point register argument
-                            intArgRegNum = min(intArgRegNum + size, MAX_REG_ARG);
+                            intArgRegNum = min(intArgRegNum + size, (unsigned int)MAX_REG_ARG);
```

Comment:
Are the casts still needed?

(If they are, it suggests the `min/max` templates aren't quite what the original code used)

Reply:
yes, that's annoying.  Without them 
```
 E:\GitHub\runtimelab\src\coreclr\jit\morph.cpp(3658): error C2672: 'min': no matching overloaded function found
  E:\GitHub\runtimelab\src\coreclr\jit\morph.cpp(27): note: could be 'const T &min(const T &,const T &)'
  E:\GitHub\runtimelab\src\coreclr\jit\morph.cpp(3658): note: 'const T &min(const T &,const T &)': template parameter 'T' is ambiguous
  E:\GitHub\runtimelab\src\coreclr\jit\morph.cpp(27): note: see declaration of 'min'
  E:\GitHub\runtimelab\src\coreclr\jit\morph.cpp(3658): note: could be 'int'
  E:\GitHub\runtimelab\src\coreclr\jit\morph.cpp(3658): note: or       'unsigned int'
  E:\GitHub\runtimelab\src\coreclr\jit\morph.cpp(3658): note: 'const T &min(const T &,const T &)': could not deduce template argument for 'const T &' from 'int'
  E:\GitHub\runtimelab\src\coreclr\jit\morph.cpp(27): note: see declaration of 'min'
```
I copied the templates from 
https://github.com/dotnet/runtimelab/blob/706c9c1ad1f4d3d150d08cd02d4ac507d02302f9/src/coreclr/inc/clr_std/utility#L53-L68

I'll have another look around, maybe there's some others.

Reply:
Ah, but `-DUSE_STL` is set so they might be from 
```
#include <utility>
```